cookbook/frontend/bower.rst

```diff
@@ -0,0 +1,125 @@
+.. index::
+    single: Front-end; Bower
+
+Using Bower with Symfony
+========================
+
+Symfony and all its packages are perfectly managed by Composer. Bower is a
+dependency management tool for front-end dependencies, like Twitter Bootstrap
+or jQuery. Since Symfony isn't purely a back-end framework, it can't deliver
```

Comment:
In this phrase there is a confusing double negation. I think it should start as `Since Symfony is purely a back-end framework ...`